source/getting-started/configuration.rst

```diff
@@ -1,9 +1,119 @@
 -------------
 
-Minimal "sane" config:
+The configuration of Unbound can be a little tricky due to the extensive array of configurable options. Below we will go through a basic, recommended config, but feel free to add and experiment with options as you need them. Also feel free to remove lines that are commented out (uncommenting bu removing the "#") if they are not nesecary in your setup.
+
+For the configuration step, we will assume that your system has a Unbound installed and it is available to the entire system (so the :command:`make install` step during installation). 
+
+The basic configuration is shown below. 
+
+.. code:: bash
+
+    server:
+        # can be uncommented if you do not need user privilige protection
+        # username: ""
+        
+        # can be uncommented if you do not need file access protection
+        # chroot: ""
+    
+        # location of the trust anchor file that enables DNSSEC. note that
+        # the location of this file can elsewhere
+        auto-trust-anchor-file: "/usr/local/etc/unbound/root.key"
+        # auto-trust-anchor-file: "/var/lib/unbound/root.key"
+    
+        # send minimal amount of information to upstream servers to enhance privacy
+        qname-minimisation: yes
+    
+        # specify the interfaces to answer queries from by ip-address.
+        interface: 0.0.0.0
+        # interface: ::0
+    
+        # addresses from the IP range that are allowed to connect to the resolver
+        access-control: 192.168.0.0/16 allow
+        # access-control: 2001:DB8/64 allow
+
+By default Unbound comes with `chroot <https://wiki.archlinux.org/title/chroot>`_ enabled. This provides an extra layer of defence against remote exploits. If :command:`chroot` gives you trouble, you can enter file paths as full pathnames starting at the root of the filesystem (``/``) or disable it in the config if this feature is not required.
+
+.. code::bash
+
+	# disable chroot
+	chroot: ""
+
+Unbound assumes that a user named "unbound" exists. You can add this user with your favourite account management tool (:command:`useradd(8)`), or disable the feature with ``username: ""`` in the config. If it is enabled, after the setup, any other user privileges are dropped and the configured username is assumed.
+
+Unbound comes with the :command:`unbound-checkconf` tool. This tool allows you to check a config for errors before loading. This tool is very convenient because if any errors are found it tells you where they are and it allows you to check this before loading Unbound to a network interface on your machine.
+
+
+Set up Remote Control
+=====================
+
+A useful functionality to enable is the use of the :command:`unbound-control` command. Enabling this allows Unbound to be controlled by using the :command:`unbound-control` command, which makes starting, stopping, and reloading Unbound easier. To enable this functionality we need to add :option:`remote-control` to the config and enable it.
```

Comment:
Duplication. Perhaps rewrite the second sentence from "Enabling this ... starting, stopping..." to "This makes it easier to start, stop and reload Unbound"?